discovery/db_discovery.go

```diff
@@ -212,3 +221,21 @@ func pmTicketParams(params *net.TicketParams) *pm.TicketParams {
 		Seed:              new(big.Int).SetBytes(params.Seed),
 	}
 }
+
+func (dbo *DBOrchestratorPoolCache) PollOrchestratorInfo(bcast common.Broadcaster) {
+	ticker := getTicker()
+	go func() {
+		for range ticker.C {
+			orchs, err := dbo.store.SelectOrchs(
+				&common.DBOrchFilter{
+					CurrentRound: dbo.rm.LastInitializedRound(),
+				},
+			)
+
+			if err != nil {
+				glog.Errorf("could not get orchestrators from database: %v", err)
```

Comment:
If we encounter an error here we should `continue`

Reply:
fixed in 9458ef19